examples/cache-handler-redis/cache-handler.js

```diff
@@ -17,7 +17,9 @@ CacheHandler.onCreation(async () => {
 
       // Redis won't work without error handling.
       client.on("error", (e) => {
```

Comment:
```suggestion
      // NB do not throw exceptions in the redis error listener,
      // because it will prevent reconnection after a socket exception.
      client.on("error", (e) => {
```